rflx/pyrflx/typevalue.py

```diff
@@ -376,7 +422,7 @@ def value(self) -> List[TypeValue]:
     @property
     def bitstring(self) -> Bitstring:
         self._raise_initialized()
-        bits: List[str] = [str(element.bitstring) for element in self._value]
+        bits = [str(element.bitstring) for element in self._value]
         return Bitstring(str.join("", bits))
```

Comment:
Would implementing `join` for `Bitstring` and removing the conversions be useful here? @treiher what's your opinion on that?

Reply:
Yes, I think a static method `join` would make sense here.

Reply:
Fixed by 7f48161